lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -1124,6 +1124,16 @@ export class StandardController implements IController {
         }
     }
 
+    /**
+     * Creates a cache interaction client to clear broswer cache.
+     * @param logoutRequest
+     */
+    async clearCache(logoutRequest?: EndSessionRequest): Promise<void> {
+        const correlationId = this.getRequestCorrelationId(logoutRequest);
+        const cacheClient = this.createSilentCacheClient(correlationId);
```

Comment:
Do we need an instance of `silentCache` or can we call `logout` independently? It is not really dependent on the client is it?

Reply:
I think the interaction clients could stand to be de-classed but I'd like to take that as its own PR. For now this is consistent with our other methods so I'm ok with it